index.js

```diff
@@ -38,73 +38,116 @@ function f(d) {
   }
   return str;
 }
-
+// TODO: store this state in a better place
+var loading = true
 async function main() {
   // Request permission
   await Notification.requestPermission();
-
   const stats = new Stats();
   stats.showPanel(0); // 0: fps, 1: ms, 2: mb, 3+: custom
-  document.getElementById('main').appendChild(stats.dom);
-  
+  // document.getElementById('main').appendChild(stats.dom);
+  if (loading) {
+    $("#timesTouchedText").hide();
+    $("#totalCount").hide();
+    $("#title").hide();
+    $("#footer").hide();
+  }
   let touchCounter = 0;
+  let totalTouches = 0;
+  let faceAlreadyTouched = false;
+  let faceCurrentlyTouched = false;
   const mobile = isMobile();
   const detectorParams = {
-    renderPointCloud: !mobile,
     width: mobile ? undefined : VIDEO_WIDTH,
     height: mobile ? undefined : VIDEO_HEIGHT,
-    timeout: 500,
-    renderPointCloud: true,
+    // renderPointCloud: !mobile, // can be used for mobile screens
+    renderPointCloud: false,
+    timeout: 300,
     renderCanvas: true,
-    renderFaceMesh: true,
+    // renderFaceMesh: true,
     onDetected: () => { touchCounter++; },
-    onRender: () => { stats.begin(); },
+    onRender: () => { 
+      stats.begin();
+      // clearInterval(loadingAnimation);
+      if (loading){
+        // let el = document.getElementById('loading-header');
+        // el.parentNode.removeChild(el);
+        $("#timesTouchedText").show();
+        $("#totalCount").show();
+        $("#title").show();
+        $("#footer").show();
+        let el = document.getElementById('loading-animation');
+        el.parentNode.removeChild(el);
+        // Copied here to ensure it only shows on load
+        const gui = new dat.GUI();
+        gui.add(state, 'frame timeout', 100, 1000).onChange((value) => { detector.update({ timeout: value }); });
+        loading = false;
```

Comment:
I think these can be move out of the onRender callback to before calling `detector.start();`